src/pages/[penname]/book/[bookId]/index.tsx

```diff
@@ -71,6 +71,7 @@ type props = InferGetServerSidePropsType<typeof getServerSideProps>;
 
 const BookContent = ({ bookId }: props) => {
   const router = useRouter();
+  const penname = router.query.penname;
```

Comment:
Get penname from getServerSideProps instead (the same as bookid)

Reply:
i'll try